web/source/keyboards/kmwkeyboards.ts

```diff
@@ -1064,14 +1065,14 @@ namespace com.keyman.keyboards {
      *
      * @param {Object}    x   metadata object
      **/
-    register(x) {
+    register(x): KeyboardStub[] {
```

Comment:
Nothing will receive the returned values.  That's why the `Promise` is there, with `resolve` and `reject`.